<html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Kill Switch | SugarGain | iGEM BITS Goa</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../../css/content.css" rel="stylesheet"></head><body><nav id="main-nav"><div class="navbar navbar-expand-lg d-flex justify-content-between bg-none nav-transparent"><a class="navbar-brand d-lg-inline-block" href="/">Sugar<span>Gain</span></a><div class="controls h-100 d-flex align-items-center"><div class="align-items-center" id="themeSwitchWrapper"><i class="icon-light"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input type="checkbox" id="themeSwitchInput"><span class="slider round"></span></label></div><i class="icon-dark"></i></div><button class="btn" id="menuSwitch"><div class="menu-icon"></div></button></div></div><div id="fullscreenMenuWrapper"><div class="w-100 h-100" id="fullscreenMenu"><div class="container-fluid d-flex align-items-stretch pr-0 flex-column"><div class="row nav-header w-100"><div class="col-xl-3 col-lg-2"><div class="teamLogo"></div></div><div class="col-xl-6 col-lg-8"></div><div class="col-xl-3 col-lg-2"></div></div><div class="row flex-grow-1 w-100"><div class="col-xl-3 col-lg-2"></div><div class="col-xl-6 col-lg-8 d-flex justify-content-between flex-column" id="menuContent"><div class="row"><div class="col-md-6" id="nav-headings"><ul role="tablist"><li><a href="javascript:;" id="Project-tab">Project</a><ul class="mobile-nav-items"><li><a href="/Description">Description</a></li><li><a href="/Design">Design</a></li><li><a href="/Model">Model</a></li><li><a href="/Engineering">Engineering</a></li><li><a href="/Safety">Safety</a></li><li><a href="/Implementation">Implementation</a></li></ul></li><li><a href="javascript:;" id="Human_Practices-tab">Human Practices</a><ul class="mobile-nav-items"><li><a href="/Human_Practices">Human-Centered Design</a></li><li><a href="/Education">Science Communication</a></li><li><a href="/Humans_of_iGEM">Humans of iGEM</a></li><li><a href="/Accessibility">Accessibility</a></li><li><a href="/All_India_iGEM_Meet">All India iGEM Meet</a></li></ul></li><li><a href="javascript:;" id="Parts-tab">Parts</a><ul class="mobile-nav-items"><li><a href="/Part_Collection">Part Collection</a></li><li><a href="/Parts">Parts</a></li></ul></li><li><a href="javascript:;" id="Journal-tab">Journal</a><ul class="mobile-nav-items"><li><a href="/Notebook">Notebook</a></li><li><a href="/Experiments">Experiments</a></li><li><a href="/Appendix">Appendix</a></li></ul></li><li><a href="javascript:;" id="Team-tab">Team</a><ul class="mobile-nav-items"><li><a href="/Team">Team</a></li><li><a href="/Attributions">Attributions</a></li><li><a href="/Collaborations">Collaborations</a></li></ul></li><li><a href="javascript:;" id="Contribution-tab">Contribution</a><ul class="mobile-nav-items"><li><a href="/Contribution">Overview</a></li><li><a href="/Software/Wiki">Wiki Software</a></li><li><a href="/Documentation">Documentation</a></li></ul></li><li><a href="javascript:;" id="Judging-tab">Judging</a><ul class="mobile-nav-items"><li><a href="/Judging/Bronze">Bronze</a></li><li><a href="/Judging/Silver">Silver</a></li><li><a href="/Judging/Gold">Gold</a></li></ul></li><li><a href="javascript:;" id="Other_Areas-tab">Other Areas</a><ul class="mobile-nav-items"><li><a href="/Excellence">Overview</a></li><li><a href="/Dance">Dance</a></li><li><a href="/Policy">Policy</a></li><li><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></li></ul></div><div class="col-6" id="nav-items"><div class="tab-content"><div class="tab-pane" id="Project-pane"><ul><li><a href="/Description">Description</a></li><li><a href="/Design">Design</a></li><li><a href="/Model">Model</a></li><li><a href="/Engineering">Engineering</a></li><li><a href="/Safety">Safety</a></li><li><a href="/Implementation">Implementation</a></li></ul></div><div class="tab-pane" id="Human_Practices-pane"><ul><li><a href="/Human_Practices">Human-Centered Design</a></li><li><a href="/Education">Science Communication</a></li><li><a href="/Humans_of_iGEM">Humans of iGEM</a></li><li><a href="/Accessibility">Accessibility</a></li><li><a href="/All_India_iGEM_Meet">All India iGEM Meet</a></li></ul></div><div class="tab-pane" id="Parts-pane"><ul><li><a href="/Part_Collection">Part Collection</a></li><li><a href="/Parts">Parts</a></li></ul></div><div class="tab-pane" id="Journal-pane"><ul><li><a href="/Notebook">Notebook</a></li><li><a href="/Experiments">Experiments</a></li><li><a href="/Appendix">Appendix</a></li></ul></div><div class="tab-pane" id="Team-pane"><ul><li><a href="/Team">Team</a></li><li><a href="/Attributions">Attributions</a></li><li><a href="/Collaborations">Collaborations</a></li></ul></div><div class="tab-pane" id="Contribution-pane"><ul><li><a href="/Contribution">Overview</a></li><li><a href="/Software/Wiki">Wiki Software</a></li><li><a href="/Documentation">Documentation</a></li></ul></div><div class="tab-pane" id="Judging-pane"><ul><li><a href="/Judging/Bronze">Bronze</a></li><li><a href="/Judging/Silver">Silver</a></li><li><a href="/Judging/Gold">Gold</a></li></ul></div><div class="tab-pane" id="Other_Areas-pane"><ul><li><a href="/Excellence">Overview</a></li><li><a href="/Dance">Dance</a></li><li><a href="/Policy">Policy</a></li><li><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></div></div></div></div><div class="row"><div id="contact-links"><ul><li><a href="https://instagram.com/igem_bits" target="_blank" rel="noopener"><i class="insta"></i><span>Instagram</span></a></li><li><a href="https://twitter.com/igembitsgoa" target="_blank" rel="noopener"><i class="twitter"></i><span>Twitter</span></a></li><li><a href="mailto:igembitsgoa@gmail.com" target="_blank" rel="noopener"><i class="email"></i><span>Email</span></a></li><li><a href="/Sample"><i class="sample"></i><span>Sample</span></a></li></ul></div></div></div><div class="col-xl-3 col-lg-2"></div></div><div class="row nav-footer w-100"><div class="col-xl-3 col-lg-2"></div><div class="col-xl-6 col-lg-8"></div><div class="col-xl-3 col-lg-2"></div></div></div></div></div></nav><style>header {
    background-image: linear-gradient(rgba(0, 0, 0, 0.4),rgba(0, 0, 0, 0),rgba(0, 0, 0, 0.4)), 
        url( ../../assets/img/Description--josh-withers.jpg);
}</style><header class="d-flex justify-content-center"><div class="container d-flex align-items-end justify-content-between"><div id="heading"><h1>Kill Switch</h1></div><a href="https://unsplash.com/photos/lZ4xZZuk8iA"><span id="bg-attribution" title="Photo by Josh Withers on Unsplash" data-placement="top"></span></a></div></header><main><div class="container"><div class="row"><div class="sidebar"><div class="nav" id="contents"><ul></ul></div></div><div class="content"><article><p>Advances in synthetic biology raise increasingly essential questions about biocontamination and biological containment. It becomes imperative that genetically-engineered organisms are not deleterious to the ecosystem and that scientists have adequate control over their production and multiplication. Certain safeguards are developed to ensure all possible safety — which are then introduced into the genetically modified organism — that compels the organism, in our case the bacteria, to rely on a “survival” factor to maintain viability in the population. This dependence can be achieved by introducing an auxotrophy for a particular compound (this is another safety aspect which we have included in our project. Visit the section on Auxotroph model for a detailed discussion about the same) or by introducing artificial systems, called Kill Switches, into the genetic material that result in cellular death under certain conditions, such as the activation of a “trigger” caused by environmental changes <a href="#citation1">Stirling et al., 2017</a>.</p><p>We intend to introduce our strain of bacteria into the sugarcane matrix to synthesise anti-invertase. Considering the close proximity of the system with both agricultural environments and human consumables, it is indispensable that our bacteria include a lethality mechanism. We propose to use a modified type II ccdA-ccdB toxin-antitoxin system as our kill switch that is activated upon exposure to atmospheric concentrations of oxygen. A change in the oxygen concentration in the environment results in a sharp increase in the amount of the toxin protein being synthesised which disturbs the already established equilibrium between the toxin and antitoxin, effectively killing the cell, and thus preventing any contamination by stray microbes. In the following sections, we explore the functioning of type II toxin-antitoxin systems, with an emphasis on the ccdA-ccdB mechanism, our rationale behind selecting this particular system, and provide a mathematical description of the processes involved.</p><h1>Type II Toxin-Antitoxin Systems</h1><p>Bacterial toxin-antitoxin systems were characterised in the 1980s as molecular systems encoded in plasmids that ensured the persistence of a plasmid in the host lineage during replication by making the cells “addicted” to the plasmid so that the only daughter cells that survived were the ones that contained the plasmid. This phenomenon is also known as postsegragational killing. Most prokaryotic chromosomes contain a number of toxin-antitoxin modules that consist of a pair of genes that encode for two components: a stable toxin protein and its unstable, labile cognate antitoxin. Toxins are always proteins, but the antitoxins can either be RNAs (in type I and III systems) or proteins (in type II systems).</p><p>Type II toxin-antitoxin systems use a protein antitoxin to deactivate the toxin protein via a protein-protein interaction. All type II antitoxins are dual-function, two-domain proteins that consist of a protein-protein interaction domain and a DNA-binding domain. The antitoxin binds to the toxin and inhibits its activity, and the stable TA complex binds to the operator of the corresponding <dfn>TAS ~ Toxin-Antitoxin System</dfn> operon via the DNA-binding domain of the antitoxin and represses its transcription <a href="#citation2">Makarova et al., 2009</a> <a href="#citation3">Tam &amp; Kline, 1989</a>.</p><p>As our product would be introduced into a plant which is later converted into a consumable item, we decided to use the ccdA-ccdB toxin-antitoxin system as the kill switch for its relative non-toxicity towards humans and more generally, mammalian cells <a href="#citation8">Reschner et al., 2013</a>.</p><h1>ccdA-ccdB Toxin-Antitoxin System</h1><p>In the ccdA-ccdB system, the two genes are organised in an operon in which the antitoxin ccdA gene is upstream of the toxin ccdB gene. ccdA prevents ccdB toxicity by forming a tight ccdA-ccdB complex <a href="#citation5">Afif et al., 2001</a>. cdB inhibits DNA replication by poisoning DNA gyrase; more specifically, ccdB prevents DNA gyrase from disentangling DNA from replication resulting in double stranded breaks in the DNA <a href="#citation6">Dao-Thi et al., 2005</a>. ccdA and ccdB are able to form complexes of various stoichiometries depending on their molar ratios, however maximal repression occurs at equimolar ratios of ccdA and ccdB <a href="#citation7">Syed et al., 2012</a>.</p><p>Our kill switch is a modified version of the ccdA-ccdB TA system, where the ccdB (toxin) <dfn>CDS ~ Coding Sequence</dfn> is placed under a T7 constitutive promoter, and the ccdA (antitoxin) <dfn>CDS ~ Coding Sequence</dfn> is placed under an FNR promoter whose activity is controlled by the concentration of oxygen present in the environment. Look at the <a href="Design#Kill_Switch">Design</a> page for a detailed description of the kill switch construct.</p><h1>Reactions Modelled and Parameter Description</h1><p>The kill switch has been designed to be activated by exposure to atmospheric concentrations of oxygen. Oxygen binds to the FNR promoter, thereby suppressing the transcription of the antitoxin and the absence of which leads to an accumulation of toxin in the cell, leading to cell death. The diagram below provides a detailed account of the processes and the reaction pathways involved.</p><blockquote><p>Image describing the construct with all its products.</p></blockquote><p>Six different chemical species have to be accounted for: the toxin mRNA (\( M_T \)), the antitoxin mRNA (\( M_A \)), the toxin (\( T \)), the antitoxin (\( A \)), a complex of the form \( TA \) (\( C_1 \)), and another complex of the form \( TAT \) (\( C_2 \)); the concentrations of these species are then expressed by forming ordinary differential equations.</p><p>Due to a difference in the time scales involved between binding/unbinding of FNR and oxygen (happening in the order of milliseconds) and transcription/translation (happening in the order of minutes), we assume that the concentration of the oxidised FNR promoter, \( [FNR-O_2] \) remains time-invariant, that is, \( \frac{d[FNR-O_2]}{dt} = 0 \). This assumption is called the <strong>quasi-steady-state approximation</strong>, which allows us to express the concentration of free promoter in terms of the concentration of oxygen and a few other parameters. We have also assumed that the rates of transcription for both the toxin and antitoxin are identical as well as that the decay rates of their respective mRNAs are the same.</p><p>We write the coupled <dfn>ODEs ~ Ordinary Differential Equations</dfn> that describe the reactions illustrated above as follows.</p><p>$$ \begin{aligned} \frac{d[M_A]}{dt} &amp;= \rho_A\frac{C_N}{1 + k_D[O_2]} - \delta_A[M_A] \\ \frac{d[M_T]}{dt} &amp;= \rho_TC_N - \delta_T[M_T] \\ \frac{d[A]}{dt} &amp;= \beta_A[M_A] + \theta_c[C_1] - d_A[A] - \alpha_c[T][A] \\ \frac{d[T]}{dt} &amp;= \beta_T[M_T] + \theta_c[C_1] + Fd_A[C_1] + 2Fd_A[C_2] - d_T[T] - \alpha_c[T][A] - \alpha_c[C_1][A] \\ \frac{d[C_1]}{dt} &amp;= \alpha_c[A][T] + \theta_c[C_2] - \theta_c[C_1] - d_T[C_1] - Fd_A[C_1] - \alpha_c[C_1][T] \\ \frac{d[C_2]}{dt} &amp;= \alpha_c[C_1][T] - \theta_c[C_2] - d_T[C_2] - Fd_A[C_2]\\ \end{aligned} $$</p><p>The parameters used in the above set of equations along with their description and numerical values are tabulated below.</p><table><caption id="table1modelparamtersandtheirnumericalvalues">Table 1: Model paramters and their numerical values</caption><thead><tr><th style="text-align:center">Parameter</th><th style="text-align:center">Description</th><th style="text-align:center">Value</th><th style="text-align:center">Units</th><th style="text-align:center">Reference</th></tr></thead><tbody><tr><td style="text-align:center">\( \rho_A \)</td><td style="text-align:center">Transcription rate of the Antitoxin</td><td style="text-align:center">\(0.1333\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \rho_T \)</td><td style="text-align:center">Transcription rate of the Toxin</td><td style="text-align:center">\(0.1333\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \beta_A \)</td><td style="text-align:center">Translation rate of the Antitoxin</td><td style="text-align:center">\(0.139\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \beta_T \)</td><td style="text-align:center">Translation rate of the Toxin</td><td style="text-align:center">\(0.033\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \delta_A \)</td><td style="text-align:center">Decay rate of the Antitoxin mRNA</td><td style="text-align:center">\(0.00203\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \delta_T \)</td><td style="text-align:center">Decay rate of the Toxin mRNA</td><td style="text-align:center">\(0.00203\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( d_A \)</td><td style="text-align:center">Decay rate of the Antitoxin</td><td style="text-align:center">\(2.8881\cdot10^{-4}\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( d_T \)</td><td style="text-align:center">Decay rate of the Toxin</td><td style="text-align:center">\(0.00115524\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( F \)</td><td style="text-align:center">Decay rate of the Antitoxin in the \( AT \) complex</td><td style="text-align:center">\(0.2\)</td><td style="text-align:center"></td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \alpha_c \)</td><td style="text-align:center">Rate of complex formation</td><td style="text-align:center">\(2\cdot10^6\)</td><td style="text-align:center">\(\text{M}^{-1}\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( \theta_c \)</td><td style="text-align:center">Rate of complex decay</td><td style="text-align:center">\(7.14972\cdot10^{-6}\)</td><td style="text-align:center">\(\text{M}^{-1}\text{s}^{-1}\)</td><td style="text-align:center"><a href="#citation9">Gelens et. al., 2013</a></td></tr><tr><td style="text-align:center">\( k_D \)</td><td style="text-align:center">Dissociation constant for FNR and \( O_2 \)</td><td style="text-align:center">\(2.5\cdot10^{-3}\)</td><td style="text-align:center">\(\text{s}^{-1}\)</td><td style="text-align:center"></td></tr></tbody></table><p>As we had no lab access, we could not verify the values of parameters through experimental analyses. The next step would be to solve the above ODEs numerically to obtain the concentrations of free toxin and antitoxin.</p><p>To activate the kill switch, we require a &quot;trigger&quot;, or a change in the surrounding oxygen concentration. This trigger was achieved by modelling the oxygen concentration as a step function, \( O(t) \), that represents the number of oxygen molecules in the environment (cell volume). $$ O(t) = \begin{cases} 3.570\times10^{-10}, &amp; \text{if $t &lt; 300$} \\ 9.735\times10^{3}, &amp; \text{if $t &gt; 300$} \end{cases} $$ The function is constructed such that we activate the kill switch at the \(t = 300 \text{ min}\) mark. This time instance was selected to allow us to observe the steady state concentrations of both, the toxin and antitoxin. The jump in the concentration corresponds to the exposure of the cell to an aerobic environment, with atmospheric levels of oxygen.</p><p>The ODEs were solved numerically using SciPy's <code>solve_ivp</code> method using the <a href="Appendix">Radau</a> algorithm and the concentrations of free ccdA and ccdB were plotted against time. We also included the concentrations of the tow complexes for good measure.</p><div class="image"><img src="../../assets/img/KillSwitch--TAGraph.jpeg" alt="Plot of the concnetrations of ccdA, ccdB and TA complexes versus time"><p>Figure 1: Plot of the concnetrations of ccdA, ccdB and TA complexes versus time</p></div><blockquote><p>The image of the graph is temporary, I'll add a better one later</p></blockquote><p>The steady-state concentration of ccdA is about \( 0.25 \text{ mM}\) . Therefore, any dynamics that lead to a higher steady state value of ccdB (which is higher than that of ccdA) would lead to an accumulation of toxin that would &quot;kill&quot; the cell.</p><p>This change is precisely brought out by changing the oxygen concentration. From the graph, it is seen that around the \(t = 300 \text{ min}\) mark, the concentration of ccdA begins to drop and that of ccdB beings to rise. After a while, the only major chemical species in the kill switch system is the toxin, ccdB. This accumulates in the cell, surpassing the stationary state concentration of ccdA, and reaches a high enough concentration to kill the cell.</p></article><div class="d-flex justify-content-between my-5"></div><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Stirling, F., Bitzan, L., O-Keefe, S., Redfield, E., Oliver, J. W. K., Way, J., &amp; Silver, P. A. (2017).</p><cite>Rational Design of Evolutionarily Stable Microbial Kill Switches.</cite><p><span class="journalTitle">Molecular Cell</span> <span class="journalInfo">68(4), 686-697.e3.</span></p><a class="in-text" href="https://doi.org/10.1016/j.molcel.2017.10.033" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Rational Design of Evolutionarily Stable Microbial Kill Switches." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Makarova, K. S., Wolf, Y. I., &amp; Koonin, E. V. (2009).</p><cite>Comprehensive comparative-genomic analysis of Type 2 toxin-antitoxin systems and related mobile stress response systems in prokaryotes.</cite><p><span class="journalTitle">Biology Direct</span> <span class="journalInfo">4(1), 19.</span></p><a class="in-text" href="https://doi.org/10.1186/1745-6150-4-19" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Comprehensive comparative-genomic analysis of Type 2 toxin-antitoxin systems and related mobile stress response systems in prokaryotes." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Tam, J. E., &amp; Kline, B. C. (1989).</p><cite>The F plasmid ccd autorepressor is a complex of CcdA and CcdB proteins.</cite><p><span class="journalTitle">Molecular and General Genetics MGG</span> <span class="journalInfo">219(1-2), 26-32.</span></p><a class="in-text" href="https://doi.org/10.1007/BF00261153" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The F plasmid ccd autorepressor is a complex of CcdA and CcdB proteins." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Fraikin, N., Goormaghtigh, F., &amp; Van Melderen, L. (2020).</p><cite>Type II Toxin-Antitoxin Systems: Evolution and Revolutions.</cite><p><span class="journalTitle">Journal of Bacteriology</span> <span class="journalInfo">202(7).</span></p><a class="in-text" href="https://doi.org/10.1128/JB.00763-19" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Type II Toxin-Antitoxin Systems: Evolution and Revolutions." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Afif, H., Allali, N., Couturier, M., &amp; Van Melderen, L. (2001).</p><cite>The ratio between CcdA and CcdB modulates the transcriptional repression of the ccd poison-antidote system.</cite><p><span class="journalTitle">Molecular Microbiology</span> <span class="journalInfo">41(1), 73-82.</span></p><a class="in-text" href="https://doi.org/10.1046/j.1365-2958.2001.02492.x" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The ratio between CcdA and CcdB modulates the transcriptional repression of the ccd poison-antidote system." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Dao-Thi, M.-H., Van Melderen, L., De Genst, E., Afif, H., Buts, L., Wyns, L., &amp; Loris, R. (2005).</p><cite>Molecular Basis of Gyrase Poisoning by the Addiction Toxin CcdB.</cite><p><span class="journalTitle">Journal of Molecular Biology</span> <span class="journalInfo">348(5), 1091-1102.</span></p><a class="in-text" href="https://doi.org/10.1016/j.jmb.2005.03.049" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Molecular Basis of Gyrase Poisoning by the Addiction Toxin CcdB." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">Syed, M. A., &amp; L vesque, C. M. (2012).</p><cite>Chromosomal bacterial type II toxin-antitoxin systems.</cite><p><span class="journalTitle">Canadian Journal of Microbiology</span> <span class="journalInfo">58(5), 553-562.</span></p><a class="in-text" href="https://doi.org/10.1139/w2012-025" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Chromosomal bacterial type II toxin-antitoxin systems." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li><li id="citation8"><p class="author">Reschner, A., Scohy, S., Vandermeulen, G., Daukandt, M., Jacques, C., Michel, B., Nauwynck, H., Xhonneux, F., Préat, V., Vanderplasschen, A. &amp; Szpirer, C. (2013).</p><cite>Use of Staby® technology for development and production of DNA vaccines free of antibiotic resistance gene.</cite><p><span class="journalTitle">Human Vaccines &amp; Immunotherapeutics</span> <span class="journalInfo">9(10), 2203-2210.</span></p><a class="in-text" href="https://doi.org/10.4161/hv.25086" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Use of Staby® technology for development and production of DNA vaccines free of antibiotic resistance gene." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext8">Back to text</a></li><li id="citation9"><p class="author">Gelens, L., Hill, L., Vandervelde, A., Danckaert, J., &amp; Loris, R. (2013).</p><cite>A General Model for Toxin-Antitoxin Module Dynamics Can Explain Persister Cell Formation in E. coli.</cite><p><span class="journalTitle">PLoS Computational Biology</span> <span class="journalInfo">9(8), e1003190.</span></p><a class="in-text" href="https://doi.org/10.1371/journal.pcbi.1003190" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=A General Model for Toxin-Antitoxin Module Dynamics Can Explain Persister Cell Formation in E. coli." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext9">Back to text</a></li></ol></article></div></div></div></main><footer><div id="footerNav"><div class="container"><div class="row justify-content-between"><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Project</a><ul><li class="py-1"><a href="/Description">Description</a></li><li class="py-1"><a href="/Design">Design</a></li><li class="py-1"><a href="/Model">Model</a></li><li class="py-1"><a href="/Engineering">Engineering</a></li><li class="py-1"><a href="/Safety">Safety</a></li><li class="py-1"><a href="/Implementation">Implementation</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Human Practices</a><ul><li class="py-1"><a href="/Human_Practices">Human-Centered Design</a></li><li class="py-1"><a href="/Education">Science Communication</a></li><li class="py-1"><a href="/Humans_of_iGEM">Humans of iGEM</a></li><li class="py-1"><a href="/Accessibility">Accessibility</a></li><li class="py-1"><a href="/All_India_iGEM_Meet">All India iGEM Meet</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Parts</a><ul><li class="py-1"><a href="/Part_Collection">Part Collection</a></li><li class="py-1"><a href="/Parts">Parts</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Journal</a><ul><li class="py-1"><a href="/Notebook">Notebook</a></li><li class="py-1"><a href="/Experiments">Experiments</a></li><li class="py-1"><a href="/Appendix">Appendix</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Team</a><ul><li class="py-1"><a href="/Team">Team</a></li><li class="py-1"><a href="/Attributions">Attributions</a></li><li class="py-1"><a href="/Collaborations">Collaborations</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Contribution</a><ul><li class="py-1"><a href="/Contribution">Overview</a></li><li class="py-1"><a href="/Software/Wiki">Wiki Software</a></li><li class="py-1"><a href="/Documentation">Documentation</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Judging</a><ul><li class="py-1"><a href="/Judging/Bronze">Bronze</a></li><li class="py-1"><a href="/Judging/Silver">Silver</a></li><li class="py-1"><a href="/Judging/Gold">Gold</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Other Areas</a><ul><li class="py-1"><a href="/Excellence">Overview</a></li><li class="py-1"><a href="/Dance">Dance</a></li><li class="py-1"><a href="/Policy">Policy</a></li><li class="py-1"><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></div></div></div></div><div id="footerBrand"><div class="container"><div class="row"><div id="footerLogo"></div></div><div class="row justify-content-center" id="footerIcons"><a class="gmail" href="mailto:igembitsgoa@gmail.com"></a><a class="insta" href="https://instagram.com/igem_bits" target="_blank" rel="noopener"></a><a class="twitter" href="https://twitter.com/igembitsgoa" target="_blank" rel="noopener"></a></div></div></div></footer><script src="../../content.bundle.js"></script></body></html>