<html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Virtual Screening | SugarGain | iGEM BITS Goa</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../../css/content.css" rel="stylesheet"></head><body><nav id="main-nav"><div class="navbar bg-none nav-transparent"><div class="row" id="navbar-row"><a class="navbar-brand" href="/"><i class="navbar-logo"></i><span> Sugar<strong>Gain</strong></span></a><div class="controls h-100 d-flex align-items-center"><div class="align-items-center" id="themeSwitchWrapper"><i class="icon-light"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input type="checkbox" id="themeSwitchInput"><span class="slider round"></span></label></div><i class="icon-dark"></i></div><button class="btn" id="menuSwitch"><div class="menu-icon"></div></button></div></div><div class="row w-100" id="progress-container-wrapper"><div id="progress-container"><div id="nav-progress-bar"></div></div></div></div><div id="fullscreenMenuWrapper"><div class="w-100 h-100" id="fullscreenMenu"><div class="container-fluid d-flex align-items-stretch pr-0 flex-column"><div class="row nav-header w-100"><div class="col-xl-3 col-lg-2"><div class="teamLogo"></div></div><div class="col-xl-6 col-lg-8"></div><div class="col-xl-3 col-lg-2"></div></div><div class="row flex-grow-1 w-100"><div class="col-xl-3 col-lg-2"></div><div class="col-xl-6 col-lg-8 d-flex justify-content-between flex-column" id="menuContent"><div class="row"><div class="col-md-6" id="nav-headings"><ul role="tablist"><li><a href="javascript:;" id="Project-tab">Project</a><ul class="mobile-nav-items"><li><a href="/Description">Description</a></li><li><a href="/Design">Design</a></li><li><a href="/Model">Model</a></li><li><a href="/Engineering">Engineering</a></li><li><a href="/Safety">Safety</a></li><li><a href="/Implementation">Implementation</a></li></ul></li><li><a href="javascript:;" id="Human_Practices-tab">Human Practices</a><ul class="mobile-nav-items"><li><a href="/Human_Practices">Human-Centered Design</a></li><li><a href="/Education">Science Communication</a></li></ul></li><li><a href="javascript:;" id="Parts-tab">Parts</a><ul class="mobile-nav-items"><li><a href="/Part_Collection">Part Collection</a></li><li><a href="/Parts">Parts</a></li></ul></li><li><a href="javascript:;" id="Journal-tab">Journal</a><ul class="mobile-nav-items"><li><a href="/Notebook">Notebook</a></li><li><a href="/Experiments">Experiments</a></li><li><a href="/Appendix">Appendix</a></li></ul></li><li><a href="javascript:;" id="Team-tab">Team</a><ul class="mobile-nav-items"><li><a href="/Team">Team</a></li><li><a href="/Attributions">Attributions</a></li><li><a href="/Collaborations">Collaborations</a></li></ul></li><li><a href="javascript:;" id="Contribution-tab">Contribution</a><ul class="mobile-nav-items"><li><a href="/Contribution">Overview</a></li><li><a href="/Software">Software</a></li></ul></li><li><a href="javascript:;" id="Entrepreneurship-tab">Entrepreneurship</a><ul class="mobile-nav-items"><li><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></li><li><a href="javascript:;" id="Judging-tab">Judging</a><ul class="mobile-nav-items"><li><a href="/Judging">Judging</a></li></ul></li></ul></div><div class="col-6" id="nav-items"><div class="tab-content"><div class="tab-pane" id="Project-pane"><ul><li><a href="/Description">Description</a></li><li><a href="/Design">Design</a></li><li><a href="/Model">Model</a></li><li><a href="/Engineering">Engineering</a></li><li><a href="/Safety">Safety</a></li><li><a href="/Implementation">Implementation</a></li></ul></div><div class="tab-pane" id="Human_Practices-pane"><ul><li><a href="/Human_Practices">Human-Centered Design</a></li><li><a href="/Education">Science Communication</a></li></ul></div><div class="tab-pane" id="Parts-pane"><ul><li><a href="/Part_Collection">Part Collection</a></li><li><a href="/Parts">Parts</a></li></ul></div><div class="tab-pane" id="Journal-pane"><ul><li><a href="/Notebook">Notebook</a></li><li><a href="/Experiments">Experiments</a></li><li><a href="/Appendix">Appendix</a></li></ul></div><div class="tab-pane" id="Team-pane"><ul><li><a href="/Team">Team</a></li><li><a href="/Attributions">Attributions</a></li><li><a href="/Collaborations">Collaborations</a></li></ul></div><div class="tab-pane" id="Contribution-pane"><ul><li><a href="/Contribution">Overview</a></li><li><a href="/Software">Software</a></li></ul></div><div class="tab-pane" id="Entrepreneurship-pane"><ul><li><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></div><div class="tab-pane" id="Judging-pane"><ul><li><a href="/Judging">Judging</a></li></ul></div></div></div></div><div class="row"><div id="contact-links"><ul><li><a href="https://instagram.com/igem_bits" target="_blank" rel="noopener"><i class="insta"></i><span>Instagram</span></a></li><li><a href="https://twitter.com/igembitsgoa" target="_blank" rel="noopener"><i class="twitter"></i><span>Twitter</span></a></li><li><a href="mailto:igembitsgoa@gmail.com" target="_blank" rel="noopener"><i class="email"></i><span>Email</span></a></li><li><a href="/Sample"><i class="sample"></i><span>Sample</span></a></li></ul><ul><li><a href="/Acknowledgements" target="_blank" rel="noopener">Acknowledgements</a></li></ul></div></div></div><div class="col-xl-3 col-lg-2"></div></div><div class="row nav-footer w-100"><div class="col-xl-3 col-lg-2"></div><div class="col-xl-6 col-lg-8"></div><div class="col-xl-3 col-lg-2"></div></div></div></div></div></nav><style>header {
    background-image: url( ../../assets/img/Description--josh-withers.jpg);
}</style><header class="d-flex justify-content-center"><div class="custom-container d-flex align-items-end justify-content-between"><div id="heading"><h1>Virtual Screening</h1></div><a href="https://unsplash.com/photos/lZ4xZZuk8iA" target="_blank" rel="noopener"><span id="bg-attribution" title="Photo by Josh Withers on Unsplash" data-placement="top"></span></a></div></header><main class="theme-project"><div class="custom-container"><div class="row"><div class="sidebar"><div class="nav" id="contents"><ul></ul></div></div><div class="content"><article><h1>Motivation</h1><p>The crux of our biosensor is the invertase inhibitor enzyme. The experiments that we have designed help us characterise this inhibitor but it was crucial that before we design our experiments, we have a sense of the different invertase inhibitors that exist and how they interact with sugarcane invertase. Although a sugarcane inhibitor would have been the first choice, while conducting a literature survey, it was found that the inhibitor from sugarcane was not extensively characterised which would prove to be a difficulty while cloning in a prokaryotic vector. Furthermore, many plant genomes harbor large, expanded gene families. These families probably originated from gene duplication events and often have evolved to fulfill diverse functions <a href="#citation1">Hothorn et al., 2010</a>. Hence, we screened potential inhibitors which have previously and extensively been cloned in <em>E. coli</em>. Prokaryotic inhibitors were not suitable as the folding and the three-dimensional structure was found to have very little homology with sugarcane invertase inhibitors.</p><h1>Procedure</h1><p>In order to perform protein-protein docking, we used the ClusPro docking server. We performed docking studies of the following plant inhibitors with invertase:</p><ol><li>Sugarcane inhibitor</li><li><em>Arabidopsis</em> inhibitor</li><li>Tobacco inhibitor</li><li>Potato inhibitor</li></ol><p>ClusPro performs three computational steps: first it performs a rigid body docking; next it performs a RMSD based clustering of the 1000 lowest energy structures; and lastly it performs a removal of steric clashes by energy minimization.</p><p>Docking methods can be classified as direct or template-based. Based on thermodynamics, direct methods attempt to find the structure of the target complex located at the minimum of Gibbs free energy in the conformational space. Template-based docking, on the other hand, is based on the observation that interacting pairs sharing above 30% sequence identity often interact in the same way, and hence the structure of the target complex can be obtained by homology modeling tools if an appropriate template complex of a known structure is available.</p><p>Although the applicability of template-based docking has been extended based on the observation that partial structures representing the interface region are capable of providing templates, the coverage of the template space at present is still limited and hence direct methods are generally more useful in many applications (ClusPro) <a href="#citation2">2</a> <a href="#citation3">3</a> <a href="#citation4">4</a>.</p><p>The interaction energy between the two proteins is given by the expression of the form $$ E = w_1E_{rep} + w_2E_{attr} + w_3E_{elec} + w_4E_{DARS} $$</p><p>where \( E_{rep} \) and \( E_{attr} \) denote the repulsive and attractive contributions to the van der Waals interaction energy, and \( E_{elec} \) is an electrostatic energy term. \( E_{DARS} \) is the DARS potential that primarily represents desolvation contributions (it is a pairwise structure-based potential constructed by the decoys as the reference state), and the coefficients \( w_1\) through \(w_4\) denote the weigths of the corresponding terms.</p><p>Initially, homology modelling was used to create the structure of sugarcane invertase and sugarcane invertase inhibitor.</p><h1>Structure of Sugarcane Invertase</h1><div class="image"><img src="../../assets/img/VS--SugarcaneInvertase.png" alt="Structure of Sugarcane invertase using homology modelling" style="width: 30%"><p>Figure 1: Structure of Sugarcane invertase using homology modelling</p></div><p>A Ramachandran plot is a way to visualize energetically favoured regions for backbone dihedral angles against amino acid residues in protein structures. Histograms with a binning of 4 degrees are used to count Φ (Phi; C-N-CA-C) / Ψ (Psi; N-CA-C-N) occurrences for all displayed categories. The number of observed Φ / Ψ pairs determines the contour lines (SWISS-MODEL) <a href="#citation5">5</a> <a href="#citation6">6</a> <a href="#citation7">7</a> <a href="#citation8">8</a> <a href="#citation9">9</a> <a href="#citation10">10</a>.</p><div class="image"><img src="../../assets/img/VS--Ramachandran Plot.png" alt="Ramachandran Plot for the modelled sugarcane invertase" style="width: 50%"><p>Figure 2: Ramachandran Plot for the modelled sugarcane invertase</p></div><p>From the above Ramachandran plot, it was observed that the Ramachandran outliers include SER169, PRO166, ASP239, SER459 and ARG387.</p><p>Next, we performed a comparision of the structure with a non-redundant set of PDB structures. The comparison with Non-Redundant Set of PDB Structures helps us derive a quality estimate for our model via QMEAN4.</p><div class="image"><img src="../../assets/img/VS--QualityComparison.png" alt="Quality comparision of the modelled sugarcane invertase" style="width: 50%"><p>Figure 3: Quality comparision of the modelled sugarcane invertase</p></div><p>Our model obtained a GMQE of 0.83. GMQE is a quality estimation score which combines properties from the target–template alignment and the template structure using a multi-layer perceptron (SWISS-MODEL) <a href="#citation5">5</a> <a href="#citation6">6</a> <a href="#citation7">7</a> <a href="#citation8">8</a> <a href="#citation9">9</a> <a href="#citation10">10</a>.</p><h1>Protein - Protein Docking</h1><p>The following images illustrate the docking of sugarcane invertase with various anti-invertases. The final energy of binding is also expressed.</p><div class="image"><img src="../../assets/img/VS--Sugarcane inhibitor.png" alt="Sugarcane invertase and sugarcane inhibitor. E = -677.5 J" style="width: 40%"><p>Figure 4: Sugarcane invertase and sugarcane inhibitor. E = -677.5 J</p></div><div class="image"><img src="../../assets/img/VS--Arabidopsis inhibitor.png" alt="Sugarcane invertase and Arabidopsis inhibitor. E = -1058.1 J" style="width: 50%"><p>Figure 5: Sugarcane invertase and Arabidopsis inhibitor. E = -1058.1 J</p></div><div class="image"><img src="../../assets/img/VS--Tobacco inhibitor.png" alt="Sugarcane invertase and Tobacco inhibitor. E = -1058 J" style="width: 50%"><p>Figure 6: Sugarcane invertase and Tobacco inhibitor. E = -1058 J</p></div><div class="image"><img src="../../assets/img/VS--Potato inhibitor.png" alt="Sugarcane invertase and Potato inhibitor E = -767.9 J" style="width: 50%"><p>Figure 7: Sugarcane invertase and Potato inhibitor E = -767.9 J</p></div><h1>Interpretation of the results</h1><p>The catalytic site of invertases is a conserved RDP domain which gets disrupted by the inhibitor. The RDP domain lies near the Asp-309 residue and contains heavy Asp and Glu residues <a href="#citation1">Hothorn et al., 2010</a>. All the inhibitors above bind to the same region. Apart from the sugarcane inhibitor itself, the Arabidopsis inhibitor shows the most disruption of the protein structure. The energy of interaction is also the most negative of all the inhibitors screened. Hence, it is feasible for us to use the inhibitor from Arabidopsis thaliana to complex with the sugarcane inhibitor for our downstream experiments as an alternative to sugarcane inhibitor.</p></article><div class="d-flex justify-content-between my-5"><a class="button prev" href="/Model/Kill_Switch"><span class="button-background"><span class="circle" aria-hidden><span class="icon arrow"></span></span></span><span class="button-text">Kill Switch</span></a><a class="button next" href="/Model/Growth_Model"><span class="button-background"><span class="circle" aria-hidden><span class="icon arrow"></span></span></span><span class="button-text">Growth Model</span></a></div><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Hothorn, M., Van den Ende, W., Lammens, W., Rybin, V., &amp; Scheffzek, K. (2010).</p><cite>Structural insights into the pH-controlled targeting of plant cell-wall invertase by a specific inhibitor protein.</cite><p><span class="journalTitle">Proceedings of the National Academy of Sciences</span> <span class="journalInfo">107(40), 17427-17432.</span></p><a class="in-text" href="https://doi.org/10.1073/pnas.1004481107" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Structural insights into the pH-controlled targeting of plant cell-wall invertase by a specific inhibitor protein." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Vajda, S., Yueh, C., Beglov, D., Bohnuud, T., Mottarella, S. E., Xia, B., ... &amp; Kozakov, D. (2017).</p><cite>New additions to the ClusPro server motivated by CAPRI.</cite><p><span class="journalTitle">Proteins: Structure, Function, and Bioinformatics</span> <span class="journalInfo">85(3), 435-444.</span></p><a class="in-text" href="https://doi.org/10.1002/prot.25219" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=New additions to the ClusPro server motivated by CAPRI." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Kozakov, D., Hall, D. R., Xia, B., Porter, K. A., Padhorny, D., Yueh, C., ... &amp; Vajda, S. (2017).</p><cite>The ClusPro web server for protein-protein docking.</cite><p><span class="journalTitle">Nature Protocols</span> <span class="journalInfo">12(2), 255-278.</span></p><a class="in-text" href="https://doi.org/10.1038/nprot.2016.169" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The ClusPro web server for protein-protein docking." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Kozakov, D., Beglov, D., Bohnuud, T., Mottarella, S. E., Xia, B., Hall, D. R., &amp; Vajda, S. (2013).</p><cite>How good is automated protein docking?.</cite><p><span class="journalTitle">Proteins: Structure, Function, and Bioinformatics</span> <span class="journalInfo">81(12), 2159-2166.</span></p><a class="in-text" href="https://doi.org/10.1002/prot.24403" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=How good is automated protein docking?." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R., ... &amp; Schwede, T. (2018).</p><cite>SWISS-MODEL: homology modelling of protein structures and complexes.</cite><p><span class="journalTitle">Nucleic Acids Research</span> <span class="journalInfo">46(W1), W296-W303.</span></p><a class="in-text" href="https://doi.org/10.1093/nar/gky427" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=SWISS-MODEL: homology modelling of protein structures and complexes." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Bienert, S., Waterhouse, A., de Beer, T. A. P., Tauriello, G., Studer, G., Bordoli, L., &amp; Schwede, T. (2016). The SWISS-MODEL Repository-new features and functionality. Nucleic Acids Research, 45(D1), D313-D319.</p><cite>The SWISS-MODEL Repository new features and functionality.</cite><p><span class="journalTitle">Nucleic Acids Research</span> <span class="journalInfo">45(D1), D313-D319.</span></p><a class="in-text" href="https://dx.doi.org/10.1093/nar/gkw1132" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The SWISS-MODEL Repository new features and functionality." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">Guex, N., Peitsch, M. C., &amp; Schwede, T. (2009).</p><cite>Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective.</cite><p><span class="journalTitle">ELECTROPHORESIS</span> <span class="journalInfo">30(S1), S162-S173.</span></p><a class="in-text" href="https://dx.doi.org/10.1002/elps.200900140" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li><li id="citation8"><p class="author">Studer, G., Rempfer, C., Waterhouse, A. M., Gumienny, R., Haas, J., &amp; Schwede, T. (2019). QMEANDisCo-distance constraints applied on model quality estimation. Bioinformatics, 36(6), 1765-1771.</p><cite>QMEANDisCo distance constraints applied on model quality estimation.</cite><p><span class="journalTitle">Bioinformatics</span> <span class="journalInfo">36(6), 1765-1771.</span></p><a class="in-text" href="https://dx.doi.org/10.1093/bioinformatics/btz828" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=QMEANDisCo distance constraints applied on model quality estimation." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext8">Back to text</a></li><li id="citation9"><p class="author">Benkert, P., Biasini, M., &amp; Schwede, T. (2010).</p><cite>Toward the estimation of the absolute quality of individual protein structure models.</cite><p><span class="journalTitle">Bioinformatics</span> <span class="journalInfo">27(3), 343-350.</span></p><a class="in-text" href="https://dx.doi.org/10.1093/bioinformatics/btq662" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Toward the estimation of the absolute quality of individual protein structure models." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext9">Back to text</a></li><li id="citation10"><p class="author">Bertoni, M., Kiefer, F., Biasini, M., Bordoli, L., &amp; Schwede, T. (2017).</p><cite>Modeling protein quaternary structure of homo-and hetero-oligomers beyond binary interactions by homology.</cite><p><span class="journalTitle">Scientific Reports</span> <span class="journalInfo">7(1).</span></p><a class="in-text" href="https://dx.doi.org/10.1038/s41598-017-09654-8" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Modeling protein quaternary structure of homo-and hetero-oligomers beyond binary interactions by homology." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext10">Back to text</a></li></ol></article></div></div></div></main><footer><div id="footerNav"><div class="container"><div class="row justify-content-between"><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Project</a><ul><li class="py-1"><a href="/Description">Description</a></li><li class="py-1"><a href="/Design">Design</a></li><li class="py-1"><a href="/Model">Model</a></li><li class="py-1"><a href="/Engineering">Engineering</a></li><li class="py-1"><a href="/Safety">Safety</a></li><li class="py-1"><a href="/Implementation">Implementation</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Human Practices</a><ul><li class="py-1"><a href="/Human_Practices">Human-Centered Design</a></li><li class="py-1"><a href="/Education">Science Communication</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Parts</a><ul><li class="py-1"><a href="/Part_Collection">Part Collection</a></li><li class="py-1"><a href="/Parts">Parts</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Journal</a><ul><li class="py-1"><a href="/Notebook">Notebook</a></li><li class="py-1"><a href="/Experiments">Experiments</a></li><li class="py-1"><a href="/Appendix">Appendix</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Team</a><ul><li class="py-1"><a href="/Team">Team</a></li><li class="py-1"><a href="/Attributions">Attributions</a></li><li class="py-1"><a href="/Collaborations">Collaborations</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Contribution</a><ul><li class="py-1"><a href="/Contribution">Overview</a></li><li class="py-1"><a href="/Software">Software</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Entrepreneurship</a><ul><li class="py-1"><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Judging</a><ul><li class="py-1"><a href="/Judging">Judging</a></li></ul></div></div></div></div><div id="footerBrand"><div class="container"><div class="row justify-content-center"><div id="footerTeamLogo"></div><div id="BITSLogo"></div></div><p>iGEM BITS Goa</p><p>Birla Institute of Technology and Science, Pilani</p><p>K K Birla Goa Campus</p><p>NH-17B, Sancoale, Goa, India</p><div class="row justify-content-center" id="footerIcons"><a class="gmail" href="mailto:igembitsgoa@gmail.com"></a><a class="insta" href="https://instagram.com/igem_bits" target="_blank" rel="noopener"></a><a class="twitter" href="https://twitter.com/igembitsgoa" target="_blank" rel="noopener"></a></div></div></div></footer><script src="../../content.bundle.js"></script></body></html>